src/scripts/main.js

```diff
@@ -1,3 +1,151 @@
 'use strict';
 
-// write code here
+const thead = document.querySelectorAll('thead tr th');
+const tbody = document.querySelector('tbody');
+let tbodyArray = Array.from(tbody.querySelectorAll('tr'));
+const sortStates = [];
+const buttonSubmit = document.querySelector('button');
+const inputName = document.querySelector('[data-qa="name"]');
+const inputPosition = document.querySelector('[data-qa="position"]');
+const inputOffice = document.querySelector('[data-qa="office"]');
+const inputAge = document.querySelector('[data-qa="age"]');
+const inputSalary = document.querySelector('[data-qa="salary"]');
+
+thead.forEach((item) => {
+  sortStates[item] = 'asc';
```

Comment:
The `sortStates` array is being used with `thead` elements as keys, which is not a typical use case for arrays. Consider using an object or Map to store sort states with column indices as keys.